File: src/main/java/no/nav/service/pensjon/regler/to/VilkarsprovForsorgingstilleggResponse.java
package no.nav.service.pensjon.regler.to;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

import java.util.ArrayList;
import java.util.List;

public class VilkarsprovForsorgingstilleggResponse extends ServiceResponse {
    private List<VilkarsVedtak> vedtaksliste = new ArrayList<>();
    private Pakkseddel pakkseddel = new Pakkseddel(true, true);

}

File: src/main/kotlin/no/nav/pensjon/regler/to/VilkarsprovForsorgingstilleggResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak


class VilkarsprovForsorgingstilleggResponse : ServiceResponse() {
    var vedtaksliste: List<VilkarsVedtak> = ArrayList()
}